src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -35,18 +34,22 @@ const Scatterplot = dynamic(
   { ssr: false },
 );
 
-const initialZoom = 4.00;
+const INITIAL_ZOOM = 4.00;
 const cellRadiusFromZoom = (zoom) => zoom ** 3 / 50;
 
+const EM = 16; // px
+const CELLINFO_WIDTH = 300; // px
```

Comment:
Ideally, we don't hardcode the cell info width -- it will vary, based on how long cell set names or gene names we have.

Reply:
Also, 300 is way too much: <img width="495" alt="Screenshot 2022-04-19 at 10 57 46" src="https://user-images.githubusercontent.com/65414651/163968524-7aa7fd31-fc21-416f-a5f6-384b0697fc4d.png">

Reply:
it hides a large proportion of the plots, I imagine that users won't like it

Reply:
Changed it to 200